app/forms/date_form.rb

```diff
@@ -16,10 +16,10 @@ class DateForm < ApplicationForm
   attribute :approximate, :boolean, default: false
 
   def to_s
-    EdtfSupport.to_edtf_s(year:, month:, day:, approximate:)
+    to_date&.to_fs(:edtf)
   end
 
-  def to_edtf
-    EdtfSupport.to_edtf(year:, month:, day:, approximate:)
+  def to_date
+    EdtfDate.for(year:, month:, day:, approximate:)
   end
```

Comment:
This method rename makes more sense to me for two reasons:

1. The returned object is a `Date` instance, not an EDTF instance. All `Date` instances can be EDTF-ified because the gem decorates all `Date` instances with EDTF goodness.
2. The `edtf` gem patches all `Date` instances so that they respond to `to_edtf`. Changing our method name here to something different avoids that confusion.